src/command/schema/fetch.rs

```diff
@@ -0,0 +1,55 @@
+use anyhow::Result;
```

Comment:
This file is also updated in https://github.com/apollographql/rover/pull/50/files#diff-a3d96ce2322c31552b8ebaf2f6ace8b71024b2070d5eca6a17c99c01904128d9

Reply:
please forgive me if there are comments on this file that we addressed in the other PR - just dismiss them :smile_cat: